cypress/e2e/issue-list.cy.ts

```diff
@@ -49,6 +45,81 @@ describe("Issue List", () => {
         });
     });
 
+    // it("renders the correct numEvents and numUsers", () => {
+    //   cy.fixture("issues-page-1.json").then((issues) => {
+
+    //     cy.log("Length of issues: ", issues.length);
+    //     cy.log("Issues: ", issues);
+
+    //     cy.get("table")
+    //       .find("tr:not(.issue-list_headerRow__3lQDP)")
+    //       .then(($rows) => {
+    //         $rows.each((index, $row) => {
+    //           cy.log("Current index:", index);
+
+    //           cy.wrap($row)
+    //             .find("td")
+    //             .first()
+    //             .should("contain", issues[index].numEvents);
+
+    //           cy.wrap($row)
+    //             .find("td")
+    //             .eq(1)
+    //             .should("contain", issues[index].numUsers);
+    //         });
+    //       });
+    //   });
+    // });
```

Comment:
I'd remove comments before merging. Here I'm not sure why there's two version of the test actually.